velox/docs/functions/presto/datetime.rst

```diff
@@ -24,6 +24,14 @@ Date and Time Functions
 
     Returns ``timestamp`` as a UNIX timestamp.
 
+.. function:: current_timestamp() -> timestamp with time zone
```

Comment:
These functions are added in alphabetical order in this file. Please move these functions to their correct order position.